examples/getting_started/OpenBotEnv.py

```diff
@@ -0,0 +1,99 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import gym
```

Comment:
Let's move this file into an `examples/common` folder and call it `openbot_env.py` for consistency with the existing `env.py` file. The rule "examples are totally self-contained except for the `common` folder" is a less confusing convention than "examples can depend on each other whenever we feel like it". Also, it is arbitrary that `OpenBotEnv` would live with `getting_started` instead of `imitation_learning_openbot`, so it is better to put it in `common` and then use a consistent hack in both examples to be able to import it.

Reply:
Updated this.